templates/my.cnf.j2

```diff
@@ -2,10 +2,18 @@
 #password = your_password
 port = {{ mysql_port }}
 socket = {{ mysql_socket }}
+{% if ansible_os_family == 'Debian' %}
+default-character-set = {{ mysql_default-character-set }}
+{% endif %}
 
 [mysqld]
 port = {{ mysql_port }}
 bind-address = {{ mysql_bind_address }}
+{% if ansible_os_family == 'Debian' %}
+character-set-client-handshake = {{ mysql_character-set-client-handshake }}
```

Comment:
Same for these—if they work with MySQL 5.1 and/or 5.5 (as well as 5.7), then they don't need to be in an if. Otherwise, we should base on version/supported features rather than OS.